x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/advanced_step/advanced_step_form.tsx

```diff
@@ -361,7 +364,43 @@ export const AdvancedStepForm: FC<CreateAnalyticsStepProps> = ({
             />
           </EuiFormRow>
         </EuiFlexItem>
-      </EuiFlexGroup>
+        <EuiFlexItem>
+          <EuiFormRow
+            label={i18n.translate('xpack.ml.dataframe.analytics.create.maxNumThreadsLabel', {
+              defaultMessage: 'Maximum number of threads',
+            })}
+            helpText={i18n.translate('xpack.ml.dataframe.analytics.create.maxNumThreadsHelpText', {
+              defaultMessage:
+                'The maximum number of threads to be used by the analysis. The default value is 1',
```

Comment:
I think this message should have a period at the end for consistency with the rest